ATeam-FantasyGame/Location.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Location.h"
+
+//Location class
```

Comment:
Good refactor,
In general, classes should have a block comment describing the scope & behavior of the class
including the functionality currently supported.